docs/language/values-and-types.mdx

```diff
@@ -1079,6 +1079,22 @@ are available for both variable-sized and fixed-sized or variable-sized arrays.
   let containsKitty = numbers.contains("Kitty")
   ```
 
+- `cadence•fun indexOf(_ element: T): Int?`
```

Comment:
I'd recommend renaming this function to `firstIndex(of: T): Int?`, [like in Swift](https://developer.apple.com/documentation/swift/array/2994720-firstindex)

Reply:
good point, will do.